src/test/java/core/basesyntax/model/OperationTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class OperationTest {
```

Comment:
according to the task, only services should be tested, but I think you can leave the tests for Operation class if the mentors are not opposed